### DISCUSSION TITLE
Guests Unable to See Images

### DISCUSSION THREAD

**Discussion by tacksupport** [requarks/wiki#5663] [Help / Questions] [answered]
**Host:**
OS: Proxmox 7 and Docker
Wiki.js Version: 2.5.287
Database engine: PostgreSQL 14.5

**Expected Behavior:**
For any path, the Guests Group should be able to see images since they have the "read:pages" and "read:assets" permissions set both in Permissions and Page Rules settings.

**As Experienced Behavior:**
Guests are able to read these pages, but are unable to see images.

**Additional Information:**
Non Guests _are_ able to see these same images as long as they are logged in.
This issue has persisted through updates to all all software layers (OS, Docker, Image, Database).

![page-rules](https://user-images.githubusercontent.com/46050351/190882667-fccec2a0-5ef6-4591-bbda-398e86c7d3f5.png)

![permissions](https://user-images.githubusercontent.com/46050351/190882668-782a7f84-fadf-4875-bd8b-6707bb7e5d5f.png)

**Comment by NGPixel** [comment, marked as answer]
Are your images under a directory named "Public-Resources"?

**Comment by tacksupport** [comment]
No, they are under the root assets folder. I have not read anything that implied or stated that would affect anything, as long as the page on which the asset appears has the correct permissions. And I have yet to find any settings that would suggest as much either.

**Reply by NGPixel** [reply]
Page rules govern both pages and assets. If you have assets at path `/foo/bar`, then you need a page rule with permission `read:assets` for the path starting with `/foo/bar`.

**Reply by tacksupport** [reply]
That did it! Thanks. I guess I just didn't link the two in my head. And silly me wrote my paths with capital letters, which assets do not support. So I will either need to rename and adjust my paths, or have separate asset folders (which I did for now as a quick fix). Much appreciated!
